Header header

int8 numOfSV

bool newPosition
bool newClockThisSolution
bool newHorizontalThisSolution
bool newHeightThisSolution
bool usesLeastSquaresPosition
bool usesFilteredL1

bool isDifferential
bool isPhase
bool isFixedInteger
bool isOmnistar
bool isStatic
bool isNetworkRTK
bool isLocationRTK
bool isBeaconDGPS

int8 initCounter
